common/src/main/java/com/microsoft/identity/common/internal/ui/webview/EmbeddedWebViewAuthorizationStrategy.java

```diff
@@ -71,11 +94,10 @@ public EmbeddedWebViewAuthorizationStrategy(@NonNull Context applicationContext,
      * The activity result is set in Authorization.setResult() and passed to the onActivityResult() of the calling activity.
      */
     @Override
-    public Future<AuthorizationResult> requestAuthorization(GenericAuthorizationRequest authorizationRequest,
-                                                            GenericOAuth2Strategy oAuth2Strategy) throws UnsupportedEncodingException {
-        mAuthorizationResultFuture = new ResultFuture<>();
-        mOAuth2Strategy = oAuth2Strategy;
-        mAuthorizationRequest = authorizationRequest;
+    @SuppressWarnings("unchecked")
```

Comment:
Aren't we suppressing warnings `unchecked` here, do we still need to fill in all the generics here ?